public/static/js/admin.js

```diff
@@ -87,6 +88,28 @@ mciModule.controller('AdminSettingsController', ['$scope','$window', 'mciAdminRe
       return;
     }
 
+    try {
+      $scope.Settings.container_pools.pools = jsyaml.safeLoad($scope.tempContainerPools);
+    } catch(e) {
+      alert("Error parsing container pools yaml: " + e);
```

Comment:
Hmm, do we have anything better than alerts? @john-m-liu

Reply:
We have a poorly named "notifier" which puts an inline warning at the top. It's being used to display server validation errors and can also be used for this

Reply:
Dropping the alerts in favor of using the "notifier"